app/webpack/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.UnitPrice.js

```diff
@@ -97,9 +97,15 @@
     // needs to be usable by cocoon:after-insert so can bind to one or many elements
     feeQuantityChange: function ($el) {
       const self = this
-      const $els = $el || $('.js-fee-quantity')
+      const $miscEls = $el || $('#misc-fees')
+      const $fixedEls = $el || $('.js-fee-quantity')
```

Comment:
If this is being used in two (or more) different places then can it be made more general by using a new class on the wrapping element instead of using the `misc-fees` id? This can be added both to the `misc-fees` element and the similar element in the other pages where this is used. It would be good to use a name beginning with `js-` so that it can be seen that it relates to Javascript.

If the two places where this happens need to be handled differently then perhaps there needs to be a new function instead of having a single function that does both.

Reply:
With the first part, we can't use one shared wrapping element as we need slightly different behaviours on the two forms. The first form we've not been working on is more automated and needs the ability to update on the original change behaviour as far as we can see. The misc fees form creates an infinite loop if we do that.

As to the second part, good point :-) Just pushing in a new commit now that splits the two into two different functions.